.taskcluster.yml

```diff
@@ -16,11 +16,14 @@ tasks:
                 $if: 'event.sender.login == "bors[bot]"'
                 then: 'skaspari+mozlando@mozilla.com'   # It must match what's in bors.toml
```

Comment:
Should this be removed while we're here?

Reply:
No yet. I thought so but bors is still used to prune old branches. I plan to get rid of it soon 🙂